recipes/unifuncnet/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "unifuncnet" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/UniFuncNet/archive/refs/tags/{{ version }}.tar.gz
+  sha256: db3a4b803404751fda7d2c82959bca0405593f885292083b1f3db7165278236f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  entry_points:
+    - unifuncnet=unifuncnet.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+    - setuptools >=42
+    - wheel
+  run:
+    - beautifulsoup4 
+    - lxml 
+    - python >=3.9
+    - requests
+    
+test:
+  imports:
+    - unifuncnet
+  commands:
+    - pip check
```

It would be nice to test the entry point:
```suggestion
    - pip check
    - unifuncnet --help
```